dummy-files/Sensors/battery_sensor.py

```diff
@@ -0,0 +1,26 @@
+import sensordata
+
+
+class BatterySensor(sensordata.SensorData):
+    """
+    Parameters
+    ----------
+    soc : float
+            State of charge.
+    u_out_inst : float
+            Instantaneous output voltage of the battery.
+    i_out_inst : float
+            Instantaneous output current of the battery.
+    temp_max : float
+            Maximum temperature of the battery pack.
+    temp_min : float
+            Minimum temperature of the battery pack.
+    """
+
+    def __init__(self, frame, timestamp, location, rotation, soc, u_out_inst, i_out_inst, temp_max, temp_min):
```

Comment:
Have you consider using [typing](https://docs.python.org/3/library/typing.html) Instead of writting Docstrings?

I think for python3.5 <= onwards this is preferred.

something like:
```python
self __init__(self, frame: Frame, ..., soc: float, temp_max: float...):
```

Reply:
I did not consider it, but can also do it

Reply:
what are the benefits?

Reply:
- Easier to mantain. You don't have to change the code and the stringdoc just the code.
- better autocomplete. Your editor will be able for example to suggest you the fields/methods of the class if you specify `location: Location`

[This article](https://florimond.dev/en/posts/2018/07/why-i-started-using-python-type-annotations-and-why-you-should-too/) summarize the whole purpose of it